orangecontrib/timeseries/widgets/owdifference.py

```diff
@@ -96,27 +114,37 @@ def commit(self):
         invert = self.invert_direction
         shift = self.shift_period
         order = self.diff_order
+        op = self.chosen_operation
+        
         for var in self.selected:
             col = np.ravel(data[:, var])
 
             if invert:
                 col = col[::-1]
 
             out = np.empty(len(col))
-            if shift == 1:
+            if op == self.Operation.DIFFERENCE and shift == 1:
                 out[:-order] = np.diff(col, order)
                 out[-order:] = np.nan
             else:
-                out[:-shift] = col[shift:] - col[:-shift]
+                if op == self.Operation.DIFFERENCE:
+                    out[:-shift] = col[shift:] - col[:-shift]
+                else:
+                    out[:-shift] = np.divide(col[shift:], col[:-shift])
+                    if op == self.Operation.PERCENT:
+                        out = (out -1) * 100
```

Comment:
Even calling it percents, nobody[†](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.pct_change.html) expects it 0..100, really. 2¢

Reply:
I usually don't bother multiplying with 100, but that always causes confusion when showing the results to other people.

Reply:
Ah, yes, the lay people, completely unfamiliar with any sort of technical field.